1_fetch/src/fetch_functions.R

```diff
@@ -0,0 +1,15 @@
+# Mackenzie Marti (mmarti-usgs, mmarti@usgs.gov)
+#
+# This script defines functions related to fetching data.
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+# retrieve data from ScienceBase
+retrieve_SB_data <- function(outPath) {
```

Comment:
Another tip for later on is that it is always good to have the explicit return() statement for all functions specifying what you are returning. This will help you make more sense of the pipeline and build the pipeline later!